tajo-plan/src/main/java/org/apache/tajo/plan/rewrite/rules/FilterPushDownRule.java

```diff
@@ -210,6 +210,20 @@ public LogicalNode visitJoin(FilterPushDownContext context, LogicalPlan plan, Lo
       }
     }
 
+      // non-equi filter should not be push down as a join qualifier until theta join is implemented
```

Comment:
It would be better if you note that this line must be restored when https://issues.apache.org/jira/browse/TAJO-742 is resolved.
The ```TODO``` annotation will be helpful for the future restoration.